specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json

```diff
@@ -1185,6 +1360,10 @@
           "type": "boolean",
           "readOnly": true,
           "description": "True if the key's lifetime is managed by key vault. If this is a key backing a certificate, then managed will be true."
+        },
+        "release_policy": {
+          "$ref": "#/definitions/KeyReleasePolicy",
+          "description": "Key release policy."
```

Comment:
Please be more descriptive. The description for `exportable` above is a good example. Remember, this is all public documentation: in the REST API docs, the doc comments for generated code in the SDK, etc.

Reply:
Fixed